src/tiled/mainwindow.cpp

```diff
@@ -310,6 +314,8 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     connect(mUi->actionZoomNormal, SIGNAL(triggered()), SLOT(zoomNormal()));
     connect(mUi->actionFullScreen, &QAction::toggled, this, &MainWindow::setFullScreen);
 
+    connect(mUi->menuCommand, SIGNAL(aboutToShow()), mCommandManager, SLOT(populateMainWindowMenu()));
```

Comment:
You can do this instead, to avoid the need for `setMainWindowMenu` earlier:
```c++
connect(mUi->menuCommand, &QMenu::aboutToShow,
        mCommandManager, [this]() { mCommandManager->populateMenu(mUi->menuCommand); });
```